### ISSUE TITLE
See which ip addresses are requesting envkey

### ISSUE THREAD

**Issue by gdi2290** [envkey/envkey-app#14]
since envkey is only used on the server it would be great to see in the app which ip addresses are requesting envkey and have a button to quickly add/block them.

**Comment by danenania** [comment]
Yep, definitely. The last release introduced EnvKey Firewall which is a step in this direction: you can now whitelist IPs and IP ranges for both server and dev keys (as well as access to the app itself). IP logging is in the works, so it should be pretty straightforward to later merge them together along these lines.

**Comment by danenania** [comment]
Both these IP audit logging and trusted IP/CIDR lists are now live in EnvKey v2, which has finally launched: https://github.com/envkey/envkey
